### PR TITLE
[Doc Improvement] [Duplicate images removed ph4]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-botbuilder-linkunfurling.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3307e3bdd631474fa0fd4b9869a94228fb119829/msteams-platform%2Fsbs-botbuilder-linkunfurling.yml) | 83 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e6bb307b-900a-4b06-ba8d-deca8017123f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-commandbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3307e3bdd631474fa0fd4b9869a94228fb119829/msteams-platform%2Fsbs-gs-commandbot.yml) | 90 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6b9b4143-c720-428a-af3e-746659fe9456) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-javascript.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3307e3bdd631474fa0fd4b9869a94228fb119829/msteams-platform%2Fsbs-gs-javascript.yml) | 90 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ff2d84ef-84dd-4c62-b596-b4b57a0367a7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-botbuilder-linkunfurling.yml [hunk 1]
```diff
@@ -99,7 +99,7 @@ items:
 
     1. Select **Review + create**.
 
-       :::image type="content" source="~/assets/images/meeting-token-generator/register-bot-3.png" alt-text="Create Microsoft App ID" border="true":::
+       :::image type="content" source="~/assets/images/Tab-ME-SSO/register-bot-2.png" alt-text="Create Microsoft App ID" border="true":::
```
Comment (ashishguptaiitb) [inline msteams-platform/sbs-botbuilder-linkunfurling.yml:102]: Change folder path to lowercase.

File: msteams-platform/sbs-gs-javascript.yml [hunk 2]
```diff
@@ -195,7 +195,7 @@ items:
 
     1. Select your **Workspace folder** by choosing your required folder.  
 
-        :::image type="content" source="../msteams-platform/assets/images/teams-toolkit-v2/first-tab/select-folder.png" alt-text="select-folder" border="false":::
+        :::image type="content" source="../msteams-platform/assets/images/teams-toolkit-v2/select-folder.png" alt-text="select-folder" border="false":::
```
Comment (ashishguptaiitb) [inline msteams-platform/sbs-gs-javascript.yml:198]: Remove border param.

File: msteams-platform/sbs-gs-commandbot.yml [hunk 3]
```diff
@@ -148,7 +148,7 @@ items:
 
     1. Select **Create a new Teams app** option to create an app using Teams Toolkit.
 
-        :::image type="content" source="../msteams-platform/assets/images/sbs-command-bot/create-new-app.png" alt-text="Wizard starts for creating a new project" border="true":::
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/create-new-app.png" alt-text="Wizard starts for creating a new project" border="true":::
```
Comment (ashishguptaiitb) [inline msteams-platform/sbs-gs-commandbot.yml:151]: Remove border param.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-botbuilder-linkunfurling.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/f07258e64c3a685c89a6b5fe80e893f5cd612f3f/msteams-platform%2Fsbs-botbuilder-linkunfurling.yml) | 83 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1d6bc33c-86a1-403c-87ea-82926d976999) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-commandbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/f07258e64c3a685c89a6b5fe80e893f5cd612f3f/msteams-platform%2Fsbs-gs-commandbot.yml) | 90 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/94c6e921-85f1-499b-8b65-9540074ecc4e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-javascript.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/f07258e64c3a685c89a6b5fe80e893f5cd612f3f/msteams-platform%2Fsbs-gs-javascript.yml) | 90 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a5b7efee-e037-492a-be11-a3a231b6620f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-botbuilder-linkunfurling.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/142d86e2774654f7e197c1bd51551522a2d889b5/msteams-platform%2Fsbs-botbuilder-linkunfurling.yml) | 83 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/48d09ca0-6183-47b2-aecd-9213565994cc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-commandbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/142d86e2774654f7e197c1bd51551522a2d889b5/msteams-platform%2Fsbs-gs-commandbot.yml) | 90 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/af0eb8d4-9a8e-43bb-a920-ddcecf6c38ef) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-javascript.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/142d86e2774654f7e197c1bd51551522a2d889b5/msteams-platform%2Fsbs-gs-javascript.yml) | 90 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/25cb7382-916c-4f96-9f26-3d87f8cc2f83) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-botbuilder-linkunfurling.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (v-sdhakshina) [reply msteams-platform/sbs-botbuilder-linkunfurling.yml]: resolved

File: msteams-platform/sbs-gs-javascript.yml (hunk 2 above)
Comment (v-sdhakshina) [reply msteams-platform/sbs-gs-javascript.yml]: border param removed

File: msteams-platform/sbs-gs-commandbot.yml [hunk 4]
```diff
@@ -148,7 +148,7 @@ items:
 
     1. Select **Create a new Teams app** option to create an app using Teams Toolkit.
 
-        :::image type="content" source="../msteams-platform/assets/images/sbs-command-bot/create-new-app.png" alt-text="Wizard starts for creating a new project":::
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/create-new-app.png" alt-text="Wizard starts for creating a new project":::
```
Comment (ashishguptaiitb) [inline msteams-platform/sbs-gs-commandbot.yml:151]: The alt-text isn't clear.

File: msteams-platform/sbs-gs-javascript.yml [hunk 5]
```diff
@@ -195,7 +195,7 @@ items:
 
     1. Select your **Workspace folder** by choosing your required folder.  
 
-        :::image type="content" source="../msteams-platform/assets/images/teams-toolkit-v2/first-tab/select-folder.png" alt-text="select-folder":::
+        :::image type="content" source="../msteams-platform/assets/images/teams-toolkit-v2/select-folder.png" alt-text="select-folder":::
```
Comment (ashishguptaiitb) [inline msteams-platform/sbs-gs-javascript.yml:198]: The alt-text is incorrect.

File: msteams-platform/sbs-botbuilder-linkunfurling.yml [hunk 6]
```diff
@@ -99,7 +99,7 @@ items:
 
     1. Select **Review + create**.
 
-       :::image type="content" source="~/assets/images/meeting-token-generator/register-bot-3.png" alt-text="Create Microsoft App ID":::
+       :::image type="content" source="~/assets/images/Tab-ME-SSO/register-bot-2.png" alt-text="Create Microsoft App ID":::
```
Comment (ashishguptaiitb) [inline msteams-platform/sbs-botbuilder-linkunfurling.yml:102]: The alt-text ends with a period. Check the best practices for alt-text in the Contributor's guide.

**Comment by ashishguptaiitb** [comment]
Hi @v-sdhakshina, I've left a few comments in the PR. Also, update from the latest and resolve conflicts, if any.

File: msteams-platform/sbs-gs-commandbot.yml (hunk 4 above)
Comment (v-habibaumme) [reply msteams-platform/sbs-gs-commandbot.yml:151]: @ashishguptaiitb,

We're editing the alt text in another PR, which we'll share soon

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-botbuilder-linkunfurling.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/ea5a1e3a580c915be6f14dbae7521c101350b2e3/msteams-platform%2Fsbs-botbuilder-linkunfurling.yml) | 83 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/93dee83c-a63b-4c2d-a3ed-e37bdb9a9d51) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-commandbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/ea5a1e3a580c915be6f14dbae7521c101350b2e3/msteams-platform%2Fsbs-gs-commandbot.yml) | 90 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/75c3072f-5f62-4471-b25e-b45dd3f5b685) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-javascript.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/ea5a1e3a580c915be6f14dbae7521c101350b2e3/msteams-platform%2Fsbs-gs-javascript.yml) | 90 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/64415d47-6d61-46d6-879d-6cdf00426c66) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-habibaumme** [comment]
@ashishguptaiitb,

Conflicts are resolved. We're working on alt text and will share the PRs soon. Request you please approve and merge this pr

File: msteams-platform/sbs-gs-javascript.yml (hunk 5 above)
Comment (v-sdhakshina) [reply msteams-platform/sbs-gs-javascript.yml:198]: Updating alt text in different PR

File: msteams-platform/sbs-botbuilder-linkunfurling.yml (hunk 6 above)
Comment (v-sdhakshina) [reply msteams-platform/sbs-botbuilder-linkunfurling.yml:102]: updating alt text in different PR

**Review by ashishguptaiitb — APPROVED** [review]

**Pull request merged**